data/intro missions.txt

```diff
@@ -451,3 +451,39 @@ conversation "end of intro missions"
 	label next
 	`	There's an awkward pause. You're unsure of how to respond. Then he says, "Thing is, this here planet is one of the most exclusive retirement communities in the galaxy. Plenty of the locals are probably worth more than a billion credits. But they're mostly financial or business types, or old money. I'm rich enough to be here, but I just hope they're willing to socialize with someone who made his fortune flying a ship."`
 	`	He hands you a credit chip worth <payment>, and says, "Best of luck to you, Captain. Don't hesitate to look me up if you're ever in the area."`
+
+
+
+mission "Intro Epilogue"
+	minor
+	landing
+	source "New Boston"
+	to offer
+		or
+			"net worth" > 100000
+			has "event: joined the free worlds"
+		not "main plot completed"
+	on offer
+		conversation
+			`The spaceport at <origin> is a completely different experience than you remember. When you first came here, you were a mere child gawking at the spaceships passing through to refuel on their way to bigger and richer worlds; when you left, you were a greenie captain, fighting tooth and nail to remain one step ahead of the bank's repossessors.`
+			branch "during main plot"
+				has "event: joined the free worlds"
+
+			`This time, however, you're an established merchant captain.`
+			`You receive a VIP's welcome in this small trading post. Merchants who shooed you away as a child are queueing up to buy your goods; young pilots are asking you for apprenticeships.`
+			`You ignore them all and walk over to the airport adjacent to the spaceport, where you rent a private jet and fly to your village. You spend a few days there, but soon you start feeling landsick. After resisting the symptoms for a few days, you finally give in, say your goodbyes and head back to your ship.`
```

Comment:
> `where you rent a private jet and fly to your village.`

I do? Just like that? With no choice or even having accepted a mission? And for free? Feels kinda weird.

Also if you're going to have the player visit their home village, there's gotta be more detail than just "You spend a few days there, but soon you start feeling landsick. After resisting the symptoms for a few days, you finally give in, say your goodbyes and head back to your ship."

I get that adding more detail would problematically define something that was previously very undefined (the player's childhood and backstory), but maybe that part _should_ stay undefined. Maybe the player doesn't need to rent a private jet and fly home for a few days to engage in unspecified activities.

Reply:
I assume the cost of a planetside jet is peanuts to an accomplished merchant captain. The mission conditions ensure that at this point the player is worth more than a quarter of the price of a brand new space ship, so they're rich by New Boston standards.

I'm open to adding a `choice` or changing the flavor story, but what could we say instead? It's hard to write this because we know next to nothing about the protagonist's history.